_rules/presentational-children-no-focusable-content-307n5z.md

```diff
@@ -0,0 +1,144 @@
+---
+id: 307n5z
+name: Element with presentational children has no focusable content
+rule_type: atomic
+description: |
+  This rule checks that elements with a role that makes its children presentational do not contain focusable elements.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:4.1.2: # Name, Role, Value (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
```

Comment:
Would this rule also map to [2.4.3 Focus order?](https://www.w3.org/TR/WCAG21/#focus-order) There is part in the understanding doc that goes:

> Focusable components need to receive focus in **an order that preserves meaning and operability** only when navigation sequences affect meaning and operability.

Although operable, elements which are not announced by the AT are meaningless so they fail the above statement.

Reply:
You are right that this effects the meaning of the focusable element, but it isn't the order that is the problem, so I don't think 2.4.3 applies here.